### DISCUSSION TITLE
Unable to reverify github education documents

### DISCUSSION THREAD

**Discussion by dev-737** [community/community#132106] [GitHub Education]
### Hi everyone,

I'm in need of help regarding renewing my GitHub Education Pack. I'm currently enrolled in the National Institute of Open Schooling (NIOS - https://nios.ac.in/) run by Central Government of India, for my 12th-grade education. Although the headquarters of my school is in another city, I attend a full-time school that is assigned as a study center in my locality by the headquarters, which is clearly stated on my ID card.

However, when I try to renew the pack, the application asks why I'm distance learning, possibly because it only recognizes the headquarters' address. This keeps leading to rejections, and I'm struggling to resolve this issue.

I'm feeling quite helpless as support hasn't responded, and really need some assistance. Any help would be greatly appreciated.

I used my ID card as document proof, and also a screenshot of the NIOS website for distance learning proof (there isn't any specific document for it since the school is only distance learning):
![image](https://github.com/user-attachments/assets/ec2b065f-8f35-4448-95d9-3c68c57f8263)
![image](https://github.com/user-attachments/assets/1a09aa23-3a71-4f3c-bb67-00336dc3bff6)

I also tried seleting my Study Center as the campus instead of NIOS but it also got rejected:
![image](https://github.com/user-attachments/assets/52a90917-62b3-4fa0-9ee7-d6b712694c8f)

Thank you so much,
dev-737

**Comment by Astrobot-me** [comment]
It kept Rejecting or did the page keep refreshing?

**Reply by dev-737** [reply]
> It kept Rejecting or did the page keep refreshing?

It kept rejecting (updated the original post with screenshots)
